app.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var question1 = 'What\'s your name?';
+var username = prompt(question1);
+console.log(question1 + ' : ' + username);
+var response1= 'Hi ' + 'username' + ', nice to meet you!';
```

Comment:
to be consistent with the rest of your code, you should have a space between `response1` and the `=`